boards/arm/nrf52840dk_nrf52840/nrf52840dk_nrf52840.dts

```diff
@@ -140,7 +140,7 @@
 };
 
 &uart0 {
-	compatible = "nordic,nrf-uart";
+	compatible = "nordic,nrf-uarte";
```

Comment:
Was that intentional? If so, it should be mentioned in the commit message, or perhaps better done in a separate commit.

Reply:
removed, it was accidentally added.